### ISSUE TITLE
kor'kron faction

### ISSUE THREAD

**Issue by deathbane1911** [HellscreamWoW/Tracker#3840]
Kor'kron faction is agressive towards horde players, some quests can't be completed due to agressive kor'kron guards one-shotting you.

**Comment by Malae1** [comment]
They are meant to be aggressive to both factions, they should only be aggressive at a certain level though.

**Comment by deathbane1911** [comment]
ok, but should they be aggressive to low levels? Some quests cannot be completed because they one-shot you, and they are generally annoying

**Comment by menelkir** [comment]
Nope, they're not mean to be aggressive, they're yellow on the official servers (you can attack them if you want, but they're not yellow at all).
